docs-sources/includes/_Requirements.md

```diff
@@ -2,12 +2,8 @@
 
 ## Node.js
 
-<aside class="notice">
-MAGE always supports the "Active LTS" (Long Term Support) versions of Node.js.
-We recommend using the most recent Active LTS version, but we always try to support the Current Release as well.
-
-You can see what the LTS and Current versions are in the Node.js
-<a href="https://github.com/nodejs/Release#release-schedule">release schedule</a>.
+<aside class="warning">
+MAGE only supports node from version 8.x
```

Comment:
Why delete the whole paragraph? 

Also, node -> Node.js

node: cli
Node.js: name of the framework